model.py

```diff
@@ -8708,7 +8708,7 @@ def cautious_http_get(cls, url, headers, **kwargs):
 
     # Sites that cause problems for us if we make automated
     # HTTP requests to them while trying to find free books.
-    AVOID_WHEN_CAUTIOUS_DOMAINS = ['gutenberg.org']
+    AVOID_WHEN_CAUTIOUS_DOMAINS = ['gutenberg.org', 'books.google.com']
```

Comment:
This is an unrelated change to make sure an OPDS import from unglue.it doesn't try to download PDFs from Google Books. (It's impossible for an automated client to get a PDF out of Google Books.)